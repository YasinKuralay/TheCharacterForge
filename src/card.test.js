import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import Card from "./card.js";

it("renders the Card component without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Card />, div);
    ReactDOM.unmountComponentAtNode(div);
});

it("matches the previous snapshot of the Card Component", () => {
    const CardComponentSnapshot = renderer.create(<Card />).toJSON();
    expect(CardComponentSnapshot).toMatchSnapshot();
});
