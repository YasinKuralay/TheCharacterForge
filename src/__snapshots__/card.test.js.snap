// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the previous snapshot of the Card Component 1`] = `
<div
  className="cardContainer"
  onMouseDown={[Function]}
  onMouseUp={[Function]}
  style={
    Object {
      "left": "650px",
      "top": "200px",
    }
  }
>
  <div
    className="innerCardContainer"
  >
    <div>
      <i
        className="fas fa-undo-alt flipSign"
        onClick={[Function]}
        onMouseDown={[Function]}
      />
      <i
        className="fas fa-times cross"
        onClick={[Function]}
      />
      <h3
        className="titleOfCard"
        onMouseDown={[Function]}
      >
        Loading
      </h3>
      <section
        className="cardBetweenBorder"
      />
      <textarea
        className="cardTextarea"
        cols="24"
        defaultValue=""
        name="frontTextArea"
        onMouseDown={[Function]}
        placeholder="You can type your content here"
        rows="18"
        spellCheck="false"
      />
    </div>
    <div
      className="backfaceOfCard"
    >
      <i
        className="fas fa-undo-alt"
        onClick={[Function]}
      />
      <h3
        className="titleOfCard"
        onMouseDown={[Function]}
      >
        NOTES: 
        Loading
      </h3>
      <textarea
        className="cardTextarea"
        cols="24"
        defaultValue=""
        name="backTextArea"
        onMouseDown={[Function]}
        placeholder="Take some notes here!"
        rows="18"
        spellCheck="false"
      />
    </div>
  </div>
</div>
`;
